app/views/shared/_changes_to_eligible_page.erb

```diff
@@ -0,0 +1,9 @@
+<br>
+<h2 class="govuk-heading-m">Content changes to the eligible page (eligibility checker)</h2>
+
+<p class="govuk-body govuk-!-font-weight-bold">Add additional information for sections:</p>
+
+<p class="govuk-body govuk-!-font-weight-regular">‘Proof that you’re recognised as a teacher’<br>(Example: ‘TRCN will charge you a fee of ₦60,000 for providing the Letter of Professional standing. They’ll also need your QTS application reference number, as well as copies of your teaching documents.’)</p>
```

Comment:
`govuk-!-font-weight-regular` isn't needed here as by default the font weight will be regular.

Reply:
Also, I think it's worth the second line being in its own paragraph.